oxide-auth/Cargo.toml

```diff
@@ -27,6 +27,9 @@ rand = "0.7.3"
 rust-argon2 = "0.8.2"
 rmp-serde = "0.14"
 url = "1.7"
+# log
+log = "0.4.8"
+log4rs = "0.10.0"
```

Comment:
I don't think those changes in `oxide-auth` were intended to be part of the commit. You problably added those in the  local version as an effort of consistent logging? A respectable effort and one that could be added to `oxide-auth` but in a separate PR :)

Reply:
Yes, I used it to trace the process locally. Sorry for this push.

Reply:
No problem. You haven't pushed the revert yet.

Reply:
Just pushed.